seDataDevices.py

```diff
@@ -214,7 +214,7 @@ def parseDevTable(self, data):
                     )
 
         if self.defnLen > devLen :
-            raise ValueError('You have defined more bytes, {}, than the message contains, {}'.format(self.defnLen, devLen))
+            log('You have defined more bytes, {}, than the message contains, {}\nYou might get some type of runtime error!\nOr just rubbish output'.format(self.defnLen, devLen))
```

Comment:
i think that raising an exception is the right thing to do here if the error is unrecoverable